src/plugins/intel_cpu/tests/functional/shared_tests_instances/low_precision_transformations/x64/fully_connected_transformation.cpp

```diff
@@ -39,11 +39,38 @@ const std::vector<ov::pass::low_precision::LayerTransformation::Params> trasform
     LayerTestsUtils::LayerTransformationParamsNGraphFactory::createParams()
 };
 
+const std::vector<FullyConnectedParams> activations = {
+    {
+        true,  // activation
+        false, // per-channel
+        true,  // FQ
+        false, // bias
+        "fullyconnected,relu_original,relu"
+    },
+    {
+        false, // activation
+        false, // per-channel
+        true,  // FQ
+        false, // bias
+        "fullyConnected_original,fullyConnected"
+    },
+    {
+        true,  // activation
+        true,  // per-channel
+        false, // FQ
+        false, // bias
+        "fullyconnected,relu_original,relu"
+    },
```

Comment:
Can you please try to add an instance with bias for x64 platform? If it will not fail in CI, then cool, we will have an extended test coverage. If fails, then we can exclude it from this PR, and investigate the failure on LPT side

Reply:
Added test instance with bias to x64.